src/lib/skilling/skills/farming/allotments.ts

```diff
@@ -0,0 +1,295 @@
+import { Plant } from '../../types';
+import itemID from '../../../util/itemID';
+
+const allotmentPlants: Plant[] = [
+	{
+		level: 1,
+		plantXp: 8,
+		checkXp: 0,
+		harvestXp: 9,
+		inputItems: { [itemID(`Potato seed`)]: 3 },
+		outputCrop: itemID('Potato'),
+		name: `Potato`,
+		aliases: ['Potato'],
+		petChance: 173_977,
+		seedType: 'allotment',
+		growthTime: 40,
+		numOfStages: 4,
+		chance1: 154,
+		chance99: 193.56,
+		chanceOfDeath: 25,
+		protectionPayment: { [itemID('Compost')]: 2 },
+		needsChopForHarvest: false,
+		fixedOutput: false,
+		givesLogs: false,
+		givesCrops: true,
+		defaultNumOfPatches: 6,
+		canPayFarmer: true,
+		canCompostPatch: true,
+		canCompostandPay: true,
+		// [QP, Patches Gained]
+		additionalPatchesByQP: [
+			[1, 2], // Canifs Patches (2)
+			[15, 3], // Harmony Island Patch (1)
+			[33, 5] // Prifddinas (2)
+		],
+		// [Farm Lvl, Patches Gained]
+		additionalPatchesByFarmLvl: [
+			[45, 2] // Farming Guide Low (2)
+		]
+	},
+	{
+		level: 5,
+		plantXp: 9.5,
+		checkXp: 0,
+		harvestXp: 10.5,
+		inputItems: { [itemID(`Onion seed`)]: 3 },
```

Comment:
use the new function we added so you dont have to do `itemID` on every key (for this whole file)